packages/server/tests/mainnetAcceptance.env

```diff
@@ -0,0 +1,7 @@
+HEDERA_NETWORK=mainnet
+CHAIN_ID=0x00
```

Comment:
should we match the mainnet chain ID 0x127?

Reply:
good catch - fixed